### PR TITLE
Prefixed persistence tables for Postgres for disambiguation between different networks

### PR DESCRIPTION
**Pull request by alexandrosfilios**

### CODE REVIEW

File: pkg/utils/errors/errors_test.go
Comment (adecaro) [inline pkg/utils/errors/errors_test.go:1]: please, could you increase unit-test coverage to 100%? It is at 70% now.

File: platform/common/core/generic/vault/txidstore/simpletxidstore.go
Comment (adecaro) [inline platform/common/core/generic/vault/txidstore/simpletxidstore.go:120]: do we still this log?

File: platform/common/core/generic/vault/txidstore/simpletxidstore.go
Comment (adecaro) [inline platform/common/core/generic/vault/txidstore/simpletxidstore.go:128]: do we still this log?

File: platform/common/core/generic/vault/txidstore/simpletxidstore.go
Comment (adecaro) [inline platform/common/core/generic/vault/txidstore/simpletxidstore.go:145]: also this?

File: platform/fabric/core/generic/committer/committer.go
Comment (adecaro) [inline platform/fabric/core/generic/committer/committer.go:85]: can't we have a logging package under common/core instead?

File: platform/view/services/db/driver/sql/common/base.go
Comment (adecaro) [inline platform/view/services/db/driver/sql/common/base.go:16]: if we import this, can we avoid to import "github.com/pkg/errors"?

**Review by adecaro — APPROVED** [review]
LGTM modulo comments. Please, apply before merging. Thanks :)

File: platform/view/services/db/driver/sql/common/base.go
Comment (alexandrosfilios) [reply platform/view/services/db/driver/sql/common/base.go:16]: There are more methods I need from `pkg/errors` unfortunately

File: platform/fabric/core/generic/committer/committer.go
Comment (alexandrosfilios) [reply platform/fabric/core/generic/committer/committer.go:85]: Sure let's align them all in a separate PR

**Pull request merged**
